jquery.flot.bubbles.js

```diff
@@ -250,7 +250,10 @@ THE SOFTWARE.
 				var y = points[iPoints - 1];
 				if (point[0] == x && point[1] == y) {
 					var radius_index = (iPoints - 2) / pointsize;
-					return parseInt(series.yaxis.scale * series.data[radius_index][2] / 2, 0);
+					let r = parseInt(series.yaxis.scale * series.data[radius_index][2] / 2, 0);
+                    if (series.bubbles.radiusCallback && (typeof (series.bubbles.radiusCallback) == "function"))
```

Comment:
Could you please also add the new option to the README so users know that it exists. Thx!